avalon/tools/loader/app.py

```diff
@@ -262,13 +264,8 @@ def closeEvent(self, event):
             print("Force quitted..")
             self.setAttribute(QtCore.Qt.WA_DeleteOnClose)
 
-        # Kill on holding SHIFT
-        modifiers = QtWidgets.QApplication.queryKeyboardModifiers()
-        shift_pressed = QtCore.Qt.ShiftModifier & modifiers
-
-        if shift_pressed:
-            print("Force quitted..")
-            self.setAttribute(QtCore.Qt.WA_DeleteOnClose)
```

Comment:
Is there a particular reason this code seems to be removed? Not crucial to me but I am just wondering whether was need to remove it since it seems unrelated to this PR.

Reply:
Ah, this part of code was actually duplicated, there are exactly same piece of code avobe this section. It looks like it was duplicated from the commit 3fea272.
And yeah, it's unrelated, just thought that it could be removed all toghter while I was adding some stuff there.